doc/deployment/deployment.md

```diff
@@ -0,0 +1,68 @@
+# DEPLOY APP
+
+## Testing & Staging: Deploy to AppCenter
+
+Download the apps:
+
+- hyperurl.co/pc-testing
+- hyperurl.co/pc-staging
+
+Or https://appcenter.ms/orgs/pass-Culture/apps/passculture-<ENV:testing|staging>-<PLATFORM:ios|android>
+
+### Soft deploy (Code Push)
+
+Most of the time, on testing, you didn't change anything in the native code. If you changed only javascript code, deploy will be **automatic** on CircleCI (deploy-soft-testing job).
+Then the build is faster as only the javascript code is published.
+
+The download and installation of the modification will be automatic when you open the app.
+
+- Troubleshoot:
+  If you don't see your changes, try to check if the codepush was well installed. To do so go to "CheatCodes", and click on the "check update" button. If it displays "no update found" you are up to date, otherwise it will donwload the update and tell you to go to AppCenter download last hard version.
```

Comment:
```suggestion
  If you don't see your changes, try to check if the codepush was correctly downloaded. To do so go to "CheatCodes", and click on the "check update" button. If it displays "no update found" you are up to date, otherwise it will download the update and tell you to go to AppCenter download last hard version.
```

Reply:
je comprends pas la fin "and tell you to go to AppCenter download last hard version" => c'est si y'a une nouvelle version ça ?

Reply:
clarifié